core/src/message/context.rs

```diff
@@ -311,3 +312,95 @@ impl MessageContext {
         (outcome, store)
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::MessageContext;
+    use crate::message::context::{ContextSettings, Error};
+
+    #[cfg(debug_assertions)]
+    #[test]
+    fn duplicated_init() {
+        let mut message_context =
+            MessageContext::new(Default::default(), Default::default(), Default::default());
+
+        assert_eq!(message_context.settings.outgoing_limit, 1024);
```

Comment:
This looks strange to me. If this value is inited with some constant so use this very same constant here. If not - then I would suggest to introduce one and adjust code & test accrodingly.

Reply:
Added constant. It should be better now